kore/src/Kore/Internal/Conditional.hs

```diff
@@ -146,54 +145,6 @@ instance TopBottom term
     isBottom Conditional {term, predicate, substitution} =
         isBottom term || isBottom predicate || isBottom substitution
 
-instance
-    ( SortedVariable variable
-    , Ord variable
-    , Show variable
-    , Unparse variable
-    , Unparse child
-    ) =>
-    Unparse (Conditional variable child)
```

Comment:
I would like to keep the structure of this instance instead of using `toTermLike`, because it highlights which parts of the output are the `term`, `predicate`, and `substitution`. What we should do here is replace `child` with `TermLike variable`, and below...